bukkit/pom.xml

```diff
@@ -336,9 +350,9 @@
             <scope>provided</scope>
         </dependency>
         <dependency>
-            <groupId>me.ryanhamshire</groupId>
+            <groupId>com.github.TechFortress</groupId>
```

Comment:
This change is wrong ^

Reply:
I followed the guide from https://github.com/TechFortress/GriefPrevention/blob/master/README.md
The forked version is the now-official repository.

Reply:
You did not follow that guide actually, you copied a small part of it and ignored the rest.

You also shouldn't be trying to follow that guide, as it doesn't apply to how Depenizen is doing things. Remove this change.